src/dotnet/UI.Blazor.App/Components/MarkupParts/UrlMarkupView.razor

```diff
@@ -4,16 +4,8 @@
     var url = Markup.Url;
 }
 
-@if (_localUrl is { } localUrl) {
-    <a class="@cls"
-       href="@(_localUrl!.Value)"
-       @onclick="@(() => _ = History.NavigateTo(localUrl))"
-       @onclick:preventDefault="true"
-       data-menu="@(MenuRef.New<MessageMenu>(Entry.Id, bool.FalseString, Markup.Url, ""))">
-        @url
-    </a>
+@if (_localUrl is not null)
```

Comment:
We don't show markup for local urls because link preview will be should below in the message view?
I suppose in some cases message text will look ugly because we swallow local links in the text. Don't you think so?

Reply:
Discussed with Alex K.